it-it/ruby-ecosystem-it.html.markdown

```diff
@@ -0,0 +1,144 @@
+---
+category: tool
+tool: ruby ecosystem
+contributors:
+    - ["Jon Smock", "http://github.com/jonsmock"]
+    - ["Rafal Chmiel", "http://github.com/rafalchmiel"]
+translators:
+    - ["Cristian Achille", "http://github.com/blackdev1l/"]
```

Comment:
i deleted the repo so i can't do that, if it will be accepted i will add it asap